Week2/muhannad.js

```diff
@@ -0,0 +1,144 @@
+// say hello world in 3 language
+console.log('Hello World'); //English
+console.log('Hola Mundo'); //Spanish
+console.log('Bonjour le monde'); //French
+
+//SyntaxError: Find a solution for this error.
+console.log("I'm awesome");
+
+//1 First, declare your variable x (do not initialize it yet).
+let x;
+//3.2 Add a console.log statement that explains in words what you think the value of x is, like in this example:
+console.log(`the value of my variable x will be undefined because it dose not have any value yet`);
+//3.3 Add a console.log statement that logs the value of x.
+console.log(x);
+//3.4 Now initialize your variable x with an integer.
+x = 11;
+//3.5 Next, add a console.log statement that explains what you think the value of x is.
+console.log('the value of x will be 11');
+//3.6 Add a console.log statement that logs the value of x.
+console.log(x);
+//4. Declare a variable y and assign a string to it.
+let y = 'hello javascript';
+//4.1 Write a console.log statement in which you explain in words what you think the value of the string is.
+console.log('the value of x will be: hello javascript');
+//4.2 Now console.log the variable y.
+console.log(y);
+//4.3 Now assign a new string to the variable y.
+y = 'i am studying javascript';
+//4.4 Just like what you did before write a console.log statement that explains in words what you think will be logged to the console.
+console.log('the value of x will be: i am studying javascript');
+//Now console.log y again.
+console.log(y);
+//How do you round the number 7.25, to the nearest integer (i.e., whole number)?
+//5.1 Declare a variable z and assign the number 7.25 to it.
+let z = 7.25;
+//5.2 console.log z.
+console.log(z);
+//5.3 Declare another variable a that has the value of z but rounded to the nearest integer.
+let a = z;
+//5.4 console.log a.
+console.log(Math.round(a));
+//5.5 So now we have z and a find a way to compare the two values and store the highest of the two in a new variable.
+//i think there are many way's to compare between 2 'built-in functions' or more value's. this is in a different way one of them.
+//5.6 console.log the highest value.
+if (a > z) {
+  console.log('bigger than b', a);
+} else {
+  console.log('z is bigger than a', z);
+}
+///////////Arrays/////////////
+//6.1 Declare an empty array. Make sure that the name you choose indicates 'plurality', because an array is capable of containing more than one element. (See Naming conventions).
+let arr1 = [, , ,];
```

Comment:
Hi Muhannad - you can also create an empty array like this: `const arr = [];`. It is similar to what you have already.

Also, please focus on using `const` keywords for creating variables wherever you can. 🙂